tests/versioned/koa-router.tap.js

```diff
@@ -123,6 +123,19 @@ tap.test('koa-router instrumentation', (t) => {
       run()
     })
 
+    t.only('should handle transaction state loss', (t) => {
```

Comment:
Should this be `t.test()` instead of `t.only()`?

Reply:
Good catch! Had that in just for testing. Will push a fix

Reply:
Fix pushed